classes/Upgrader.php

```diff
@@ -351,7 +351,7 @@ public function getXmlMd5File($version, $refresh = false)
      * returns an array of files which are present in PrestaShop version $version and has been modified
      * in the current filesystem.
      *
-     * @return array of string> array of filepath
+     * @return array|false
```

Comment:
```suggestion
     * @return string[]|false
```

Reply:
So we must prefer this syntax when we know it's only composed of strings?

Reply:
If we confirm that our analysis tools are able to understand these types, I'm in favor of using them. :)